events/views.py

```diff
@@ -28,6 +29,7 @@ def createEvent(request):
     if request.method == 'POST':
         form = EventsForm(request.POST)
         if form.is_valid():
+            event = form.save()  # This will save the event with start and end times
```

Comment:
We have created a separate api for saving so we didnt add this 

If you see from line 13-26 we have another api
you need to add changes in that as well